### PR TITLE
Make the UI team owner of the `themes` directory

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (ahus1) [inline .github/CODEOWNERS:46]: I agree that all files in here are UI team based. Still I wonder if want other maintainers to help with the translation files in `messages*.properties`. My suggestion below would allow either a UI maintainer or an overall maintainer to approve a change. 

```suggestion
/themes/                                                    @keycloak/ui-maintainers @keycloak/maintainers
```

**Review by ahus1 — COMMENTED** [review]
Hi @jonkoops - thank you for the PR. I might suggest a small tweak similar to the handling of the cloud-native-maintainers setup in the same file. 

If you think it should be kept as is, let me know.

File: .github/CODEOWNERS
Comment (jonkoops) [reply .github/CODEOWNERS:46]: Would that not require a review from both rather than either?

Comment (ahus1) [reply .github/CODEOWNERS:46]: From what I understand, having two entries would notify both groups, but only one needs to approve. This is how I recall it from the cloud native entries.

Comment (vmuzikar) [reply .github/CODEOWNERS:46]: Yes, for cloud-native I can confirm there's a logical OR. Either someone from `@keycloak/cloud-native-maintainers` approves, or a global maintainer. Which is the desired behaviour (for Cloud Native team).

Comment (jonkoops) [reply .github/CODEOWNERS]: Got it, made the change.

**Comment by jonkoops** [comment]
@ahus1 @vmuzikar made the proposed changes, can I ask for another round of reviews?

**Review by ahus1 — APPROVED** [review]

**Pull request merged**
